lib/misc/blocks.coffee

```diff
@@ -0,0 +1,56 @@
+# TODO: docstrings
+
+scopes = require './scopes'
+
+module.exports =
+
+  getLine: (ed, l) ->
+    ed.getTextInBufferRange [[l, 0], [l, Infinity]]
```

Comment:
Good point – I only recently discovered this trick and there are definitely uglier versions of it lying around.